zone/inventory.cpp

```diff
@@ -650,8 +650,8 @@ bool Client::SummonItem(uint32 item_id, int16 charges, uint32 aug1, uint32 aug2,
 
 	// put item into inventory
 	if (to_slot == EQ::invslot::slotCursor) {
-		PushItemOnCursor(*inst);
-		SendItemPacket(EQ::invslot::slotCursor, inst, ItemPacketLimbo);
+		PushItemOnCursor(*inst, true);
+		//SendItemPacket(EQ::invslot::slotCursor, inst, ItemPacketLimbo);
```

Comment:
Remove if we're not going to keep. Although we was this removed ? Is the true to PushItemOnCursor making up for this ?

Reply:
yes, it does.